app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [
+        Person(person_data["name"], person_data["age"])
+        for person_data in people
+    ]
+
+    for person in people:
+
+        if "wife" in person:
+            if person.get("wife"):
+                Person.people[person.get("name")].wife =\
```

Comment:
Rewrite this so you don't use `\` for breaking lines

Reply:
not fixed.

Reply:
Sorry, I just woke up :)